charts/consul/templates/mesh-gateway-deployment.yaml

```diff
@@ -237,7 +237,7 @@ spec:
             {{- if .Values.global.adminPartitions.enabled }}
             -service-partition={{ .Values.global.adminPartitions.name }} \
             {{- end }}
-            -log-level={{ default .Values.global.logLevel }} \
```

Comment:
was this intentional?

Reply:
Nitya mentioned above that she will remove it before merge. I have not idea why it didn't show up in the diffs for review. 🤷

Reply:
weird!